process.sh

```diff
@@ -225,27 +225,37 @@ process_mermaid() {
   # to be done on the client-side.
 
   CSV_IN="$INPUT/data.csv"
-  PNG_IN="$INPUT/bg.png"
 
   CLI_ARGS="--csv_file $CSV_IN"
   add_arg 'cells' 'mermaid'
   add_arg 'molecules' 'mermaid'
+  add_arg 'images' 'mermaid'
 
   echo "Download and process cells..."
 
   [ -e "$CSV_IN" ] || \
     curl "$MERMAID_URL/data.csv.gz" | gunzip -d > "$CSV_IN"
 
-  [ -e "$CSV_IN" ] || \
-    wget "$MERMAID_URL/bg.png" -O "$PNG_IN"
+  PNG_OUT="$OUTPUT/mermaid.png"
+  if [ -e "$PNG_OUT" ]
+  then
+    echo "Skip images -- output already exists: $PNG_OUT"
+  else
+    curl "$MERMAID_URL/bg.png" > "$OUTPUT/mermaid.png"
```

Comment:
Use `wget`
- in this case, because we have the special code to override it during tests to avoid network dependencies,
- and in general, because `curl` is really a lower level tool: It gives you a lot more control over the HTTP details, but if you just want to download, `wget` is better